src/app/api-server/src/commands/run_command.rs

```diff
@@ -0,0 +1,173 @@
+// Copyright Kamu Data, Inc. and contributors. All rights reserved.
+//
+// Use of this software is governed by the Business Source License
+// included in the LICENSE file.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0.
+
+use std::future::{Future, IntoFuture as _};
+use std::path::PathBuf;
+use std::pin::Pin;
+
+use internal_error::*;
+use kamu::domain::TenancyConfig;
+use kamu_accounts::CurrentAccountSubject;
+
+use crate::ui_configuration::UIConfiguration;
+
+////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+
+pub struct RunCommand {
+    catalog: dill::Catalog,
+
+    // TODO: Inject from catalog?
+    tenancy_config: TenancyConfig,
```

Comment:
Why not?

Reply:
I didn't want to repeat the same (flawed) "manual DI" approach as in `kamu-cli` (see my comment above), so all values are being passed to commands manually for now.